server/views/index.html

```diff
@@ -190,8 +189,18 @@
                     </div>
                 </div>
             </div> <!--close tabcontents !-->
-
-
```

Comment:
Theses removal should be squashed in the commit that introduces the new lines :)